cmake/CheckExtraCompilerFeatures.cmake

```diff
@@ -259,7 +259,11 @@ function( NEST_CHECK_RANDOM123 )
   if ( ${COMPILE_RESULT} AND ${RUN_RESULT} EQUAL 0 )
     set( HAVE_RANDOM123 ON )
   else ()
-    message( ${RUN_OUTPUT} )
+    if ("${RUN_OUTPUT}" STREQUAL "")
+      message( "Test output is empty." )
```

Comment:
Why do we need this message? Would it not be better to call `message()` only if `RUN_OUTPUT` is not empty?

Reply:
`RUN_OUTPUT` is empty only when the compilation of the Random123 tests failed, so I've added a message for that instead.